src/Core/Package.vala

```diff
@@ -254,6 +254,12 @@ public class AppCenterCore.Package : Object {
         }
     }
 
+    public bool is_flatpak {
+        get {
+            return (backend is FlatpakBackend) || change_information.updatable_packages.contains (FlatpakBackend.get_default ());
```

Comment:
Tempting to split into two lines based on the length?

Reply:
@davidmhewitt okay possibly dumb question, but: how would you prefer this line be split? There aren't wrapping parens or anything so it always looks awkward no matter how I split it. Also, other lines in this same file are longer, so I'm tempted to leave it…

Reply:
Yeah, not the easiest one to split. It may be better to start with an if statement that checks `backend is FlatpakBackend` and returns true at that point if so. And then have the longer statement checking `updateable_packages` on its own line. But I'm fine to leave this either way.